analysis/results_conversion.py

```diff
@@ -0,0 +1,232 @@
+import joint_distribution_pb2
+import distbench_pb2
+import traffic_config_pb2
+import os
+import argparse
+
+# This a script to process the results from the distbench results proto format
+# with the purpose of putting the results into a text file
+
+class DefaultFormatter:
+    # this function takes a list of rpc and converts them into an intermediate format
+    def summarize(self, rpc_list):
+        summary = []
+        for rpc in rpc_list:
+            summary.append(tuple([rpc.request_size, rpc.latency_ns]))
+        return summary
+
+    # this function takes a summary generated by the summarize function and converts it
```

Comment:
Takes datasets in the intermediate format, possibly concatenated together to ofrom high-level summaries.